goobi_adapter/goobi_reader/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <root level="info">
```

Comment:
Embrace the brave new world of #2209?

Reply:
Yep, but it's not in master yet and even in that PR is only for two apps. I'm guessing the reason for that is to try it out first so I don't really want to add it here until it's standard